test/CsvReaderTest.java

```diff
@@ -0,0 +1,39 @@
+import com.reader.AbstractReader;
+import com.reader.FactoryReader;
+import com.sudoku.SudokuBoard;
+import junit.framework.Assert;
+import org.junit.Test;
+
+/**
+ * This class represent the unit tests for the csv Reader class
+ *
+ * @author Jose Cabrera
+ */
+public class CsvReaderTest {
+
+    private static final String USER_DIR = System.getProperty("user.dir");
+
+    @Test
+    public void readCsvFile() {
+        String txtContent = "003020600900305001001806400008102900700000008006708200002609500800203009005010300";
+        AbstractReader readerCsv = FactoryReader.createReader(USER_DIR + "\\test.csv");
```

Comment:
This unit test does not run if i try to execute it, remember that unit test should run independently of the environment